hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/statistics/impl/EmptyS3AStatisticsContext.java

```diff
@@ -317,6 +322,9 @@ public long getInputPolicy() {
       return 0;
     }
 
+    @Override
+    public long getSkipOperations() { return 0; }
```

Comment:
This line introduces a new checkstyle violation, can you drop the return on to a new line?

```
./hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/statistics/impl/EmptyS3AStatisticsContext.java:326:    public long getSkipOperations() { return 0; }:37: '{' at column 37 should have line break after. [LeftCurly]
```

Reply:
@dannycjones have updated